rom/dev/src/flow/cold_reset/ldev_id.rs

```diff
@@ -173,10 +172,7 @@ impl LocalDevIdLayer {
         //cprintln!("[ldev] Erasing AUTHORITY.KEYID = {}", auth_priv_key as u8);
         env.key_vault.erase_key(auth_priv_key)?;
 
-        // Verify the signature of the `To Be Signed` portion
-        if !Crypto::ecdsa384_verify(env, auth_pub_key, tbs.tbs(), sig)? {
```

Comment:
It seems that this verify is still requested from a glitch perspective.

Reply:
Added back.